src/sort/index.ts

```diff
@@ -1,33 +1,57 @@
-import { alias, configurable, tag, Events, Selectors, Store, Tag } from '@storefront/core';
+import { alias, configurable, tag, Events, Selectors, Store, StoreSections, Tag } from '@storefront/core';
 
 @configurable
 @alias('sort')
 @tag('gb-sort', require('./index.html'))
 class Sort {
 
   props: Sort.Props = {
-    labels: []
+    labels: [],
   };
```

Comment:
A point for discussion: maybe we can set a default value for props.storeSection?

Reply:
We do, this happens in core.